adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -1888,4 +1889,29 @@ abstract class AvroGenomicRDD[T <% IndexedRecord: Manifest, U <: Product, V <: A
   def saveAsParquet(filePath: java.lang.String) {
     saveAsParquet(new JavaSaveArgs(filePath))
   }
+
+  def saveAsPartitionedParquet(filePath: String,
+                               blockSize: Int = 128 * 1024 * 1024,
```

Comment:
We should drop all the options other than compression codec and partition size, since they are ignored.